src/modules/drawing/DrawingModule.vue

```diff
@@ -249,17 +304,19 @@ export default {
         triggerKMLUpdate() {
             clearTimeout(this.KMLUpdateTimeout)
             this.KMLUpdateTimeout = setTimeout(
-                () => {
-                    const kml = generateKmlString(this.drawingLayer.getSource().getFeatures())
-                    if (kml && kml.length) {
-                        this.saveDrawing(kml)
-                    }
-                },
+                this.triggerImmediateKMLUpdate,
                 // when testing, speed up and avoid race conditions
                 // by only waiting for next tick
                 IS_TESTING_WITH_CYPRESS ? 0 : 2000
             )
         },
+        async triggerImmediateKMLUpdate() {
+            clearTimeout(this.KMLUpdateTimeout)
+            const kml = generateKmlString(this.drawingLayer.getSource().getFeatures())
+            if (kml && kml.length) {
+                await this.saveDrawing(kml)
```

Comment:
All errors while updating or creating a KML file are already logged at the source in files.api.js. So it is probably not useful to relog the same error in the vue module. Or do you think that it could still be usefull?
Also if the error happens while closing or opening the drawing mode, the error is already logged also in the vue module.

Reply:
If the API file does it, then it's all good. But if the API file can sometimes `reject()` a promise, it is better to `try...catch` and log the error

Reply:
The api catches the rejected promise, logs the error, and then rejects the promise again. Can I leave it like that or should I relog it?